addons/website/static/src/scss/website.ui.scss

```diff
@@ -4,12 +4,11 @@
 ///
 
 $-mini-nav-size: 40px;
+$-mini-nav-arrow-size: 24px;
```

Comment:
In prefer to avoid the use of arbitrary values.
When it's needed I usually create a variables to handle them, especially if it's about layouting.

```scss
.o_frontend_to_backend_apps_btn, .o_frontend_to_backend_edit_btn {
    transform: translateX(-#{$-mini-nav-arrow-size});
 }
```
It clarify that the translation value should be equal the the arrow size... I could have added a note, of course,  but this looks cleaner to me and a little bit more maintainable.